File: app/src/main/java/com/mithilakshar/mithilapanchang/calendardialog.java
package com.mithilakshar.mithilapanchang;

import android.app.Dialog;
import android.content.Context;
import android.view.Window;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

public class calendardialog extends Dialog {

    private TextView calendartext,calendartext1;
    public calendardialog(@NonNull Context context) {
        super(context);
        requestWindowFeature(Window.FEATURE_SWIPE_TO_DISMISS);
        setContentView(R.layout.calendardialog);
        calendartext=findViewById(R.id.calendardialogtext);
        calendartext1=findViewById(R.id.calendardialogtext1);


    }

    public calendardialog(@NonNull Context context, int themeResId) {
        super(context, themeResId);
    }

    protected calendardialog(@NonNull Context context, boolean cancelable, @Nullable OnCancelListener cancelListener) {
        super(context, cancelable, cancelListener);
    }

    public void setcalendardialogtext(String text){
        calendartext.setText(text);
    }
    public void setcalendardialogtext1(String text){
        calendartext1.setText(text);
    }

}

File: app/src/main/java/com/mithilakshar/mithilapanchang/calendardialog.kt
package com.mithilakshar.mithilapanchang

import android.app.Dialog
import android.content.Context
import android.content.DialogInterface
import android.view.Window
import android.widget.TextView

class calendardialog : Dialog {
    private var calendartext: TextView? = null
    private var calendartext1: TextView? = null

    constructor(context: Context) : super(context) {
        requestWindowFeature(Window.FEATURE_SWIPE_TO_DISMISS)
        setContentView(R.layout.calendardialog)
        calendartext = findViewById(R.id.calendardialogtext)
        calendartext1 = findViewById(R.id.calendardialogtext1)
    }

    constructor(context: Context, themeResId: Int) : super(context, themeResId)
    protected constructor(
        context: Context,
        cancelable: Boolean,
        cancelListener: DialogInterface.OnCancelListener?
    ) : super(context, cancelable, cancelListener)

    fun setcalendardialogtext(text: String?) {
        calendartext!!.text = text
    }

    fun setcalendardialogtext1(text: String?) {
        calendartext1!!.text = text
    }
}
